tutorials/OLD_grafana/create-dashboard-and-panel.md

```diff
@@ -20,53 +20,34 @@ and visualize data.
 Before you begin, make sure you have:
 
 *   [Installed TimescaleDB][install-timescale].
```

Comment:
Add instructions to cloud. "installed TimescaleDB or created a Timescale account (with link to sign up)."

Reply:
@jayercule, besides adding the cloud info, this is okay. Can you please make those changes, or can I go ahead and make them? I'll then just copy the md file to our blog, as agreed with Justin.